packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3268,6 +3270,7 @@ export type MessageEditedListener = MessageReceivedListener;
 export type MessageOptions = {
     metadata?: Record<string, string>;
     attachments?: AttachmentMetadata[];
+    type?: 'text' | 'html';
```

Comment:
Nit: Do we want to have a more specific naming? Like chatMessageType or messageType or contentType?

Reply:
i think we need to use type so that it matches the Chat SDK type

Reply:
We could separate it out into a named type like the Chat SDK does though

Reply:
Updated to have separate type to match Chat SDK

![image](https://github.com/Azure/communication-ui-library/assets/73612854/afabf256-d145-4c8d-808a-50c1095dff4d)